doc/dev/tests.md

```diff
@@ -5,24 +5,52 @@ title: Tests
 
 ## Overview
 
-Npgsql comes with an extensive test suite to make sure no regressions occur. All tests are run on our build server on all supported .NET versions (including a recent version of mono) and all supported PostgreSQL backends.
+Npgsql has an extensive test suite to guard against regressions. The test suite is run on the official build server for all supported .NET versions (including a recent version of mono) and all supported PostgreSQL backends.
 
-There is also a growing suite of speed tests to be able to measure performance. These tests are currently marked [Explicit] and aren't executed automatically.
+Npgsql also includes a growing suite of performance tests. These tests are marked `[Explicit]` and can be executed manually.
 
-## Simple setup
+## Getting Started
 
-The Npgsql test suite requires a PostgreSQL backend to test against. Simply use the latest version of PostgreSQL on your dev machine on the default port (5432).
-By default, all tests will be run using user *npgsql_tests*, and password *npgsql_tests*. Npgsql will automatically create a database called *npgsql_tests* and
-run its tests against this.
+The Npgsql test suite requires a PostgreSQL backend for tests to run. By default, the test suite expects PostgreSQL to be running on the local machine with the default port (5432). 
 
-To set this up, connect to PostgreSQL as the admin user as follows:
+### Install PostgreSQL
+
+#### Linux
+
+1. Install PostgreSQL: https://www.postgresql.org/download/linux
+2. Start the PostgreSQL service: `sudo service postgresql start`
+
+#### Windows Subsystem for Linux
+
+On Windows, the Npgsql test suite can be run against a PostgreSQL server running in the [Windows Subsystem for Linux](https://docs.microsoft.com/en-us/windows/wsl/about) (WSL):
+
+1. Install WSL: https://docs.microsoft.com/en-us/windows/wsl/install-win10
+2. Install PostgreSQL: https://www.postgresql.org/download/linux
+3. Start the PostgreSQL service: `sudo service postgresql start`
```

Comment:
It's possible to run PostgreSQL on Windows without any other things installed or in a Docker container. Therefore, I don't think that we should provide such information.

Reply:
Yeah, I agree with @YohDeadfall - on the rare occasion I develop on Windows, I generally use the [excellent EnterpriseDB installer](https://www.postgresql.org/download/windows/) which works out of the box.

Reply:
Adding another section for docker setup sounds useful to add, but I'm not sure I see any reason _not_ to provide this as well. 

WSL is a built-in sandbox for just this type of thing. So why not provide some options to potential contributors?